seed/data_importer/match.py

```diff
@@ -115,11 +115,21 @@ def match_and_link_incoming_properties_and_taxlots(file_pk, progress_key):
     tax_lot_initial_incoming_count = incoming_tax_lots.count()
 
     if incoming_properties.exists():
+        # If importing BuildingSync, we will not just skip duplicates like we normally
+        # do. Since we don't skip them, they will eventually get merged into their "duplicate".
+        # We do this b/c though the property data might be the same, the Scenarios, Measures,
+        # or Meters might have been updated. The merging flow is able to "transfer"
+        # this data, while skipping duplicates cannot.
```

Comment:
maybe this goes in the docstring?

Reply:
Hmm, I could go either way. I think since this is more implementation specific a user using the function shouldn't need to be aware of this, and I kind of like that it's close to the code it's relevant to